src/CoreBundle/Service/MaintenanceModeServiceInterface.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace ChameleonSystem\CoreBundle\Service;
+
+use ChameleonSystem\CoreBundle\Exception\MaintenanceModeErrorException;
+
+interface MaintenanceModeServiceInterface
+{
+    /**
+     * Returns if the maintenance mode is activated (by file).
+     * Method should not query the database.
+     *
+     * @return bool
+     */
+    public function isActivated(): bool;
+
+    /**
+     * Returns if the maintenance mode is activated (in database).
```

Comment:
As written above I would change the method name and then also the description.

Reply:
Removed from interface for now: will be changed for #119